evap/staff/templates/quick-review.js

```diff
@@ -29,6 +32,7 @@ $(document).ready(() => {
             "k":            "[data-action=make_private]",
             "l":            "[data-action=hide]",
             "backspace":    "[data-action=unreview]",
+            "e":            "[data-action=textanswer_link]",
```

Comment:
this action should probably better be called `textanswer_edit` (here and everywhere else)